internal/cmdliveinit/cmdliveinit.go

```diff
@@ -5,37 +5,49 @@ package cmdliveinit
 
 import (
 	"context"
-	"crypto/sha1"
+	goerrors "errors"
 	"fmt"
+	"io/ioutil"
 	"os"
-	"strconv"
+	"path/filepath"
 	"strings"
-	"time"
 
 	"github.com/GoogleContainerTools/kpt/internal/docs/generated/livedocs"
 	"github.com/GoogleContainerTools/kpt/internal/errors"
 	"github.com/GoogleContainerTools/kpt/internal/pkg"
 	"github.com/GoogleContainerTools/kpt/internal/printer"
+	"github.com/GoogleContainerTools/kpt/internal/types"
 	"github.com/GoogleContainerTools/kpt/internal/util/attribution"
 	kptfilev1 "github.com/GoogleContainerTools/kpt/pkg/api/kptfile/v1"
+	rgfilev1alpha1 "github.com/GoogleContainerTools/kpt/pkg/api/resourcegroup/v1alpha1"
 	"github.com/GoogleContainerTools/kpt/pkg/kptfile/kptfileutil"
 	"github.com/spf13/cobra"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 	"sigs.k8s.io/cli-utils/pkg/common"
 	"sigs.k8s.io/cli-utils/pkg/config"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
 const defaultInventoryName = "inventory"
 
 // InvExistsError defines new error when the inventory
-// values have already been set on the Kptfile.
+// values have already been set on the ResourceGroup file.
 type InvExistsError struct{}
 
 func (i *InvExistsError) Error() string {
 	return "inventory information already set for package"
 }
 
+// InvInKfExistsError defines new error when the inventory
+// values have already been set on the Kptfile and we will warn
+// the user to migrate rather than init.
+type InvInKfExistsError struct{}
+
+func (i *InvInKfExistsError) Error() string {
+	return "inventory information already set within Kptfile for package"
```

Comment:
this error is ambiguous with `InvExistsError` in ln38-40

Reply:
This error is specific to handling `kpt live init` when the inventory is present in the Kptfile. The `InvExistsError` in ln38-40 refers to when inventory exists in a resourcegroup file.